website/src/app/[lang]/[region]/(website)/open-source/(components)/contributors.tsx

```diff
@@ -0,0 +1,57 @@
+const owner = 'socialincome-san';
+const repo = 'public';
+
+interface Contributor {
+	id: number;
+	name: string;
+	commits: number;
+	avatarUrl: string;
+}
+
+interface GitHubContributor {
+	author: {
+		id: number;
+		login: string;
+		avatar_url: string;
+	};
+	total: number;
+}
+
+export async function getContributors(): Promise<Contributor[]> {
+	const res = await fetch(`https://api.github.com/repos/${owner}/${repo}/stats/contributors`);
```

Comment:
Please update this fetch request to use the following header:

```ts
`Authorization: Bearer ${process.env.GITHUB_PAT}`
```

This hopefully fixes the issue that the request didn't work on the deployment branch.

```ts
const res = await fetch(`https://api.github.com/repos/${owner}/${repo}/stats/contributors`, {
  headers: {
    'Authorization': `Bearer ${process.env.GITHUB_PAT}`,
    'Accept': 'application/vnd.github+json'
  }
});
```

Reply:
To test this locally, you might need to create a token yourself and add it to the .env file.

You can create a token here: https://github.com/settings/tokens?type=beta

Reply:
Thank you for the help @mkue